ocs/defaults.py

```diff
@@ -32,6 +32,8 @@
 CEPH_IMAGE = "ceph/ceph:v14.2.0-20190410"
 ROOK_IMAGE = "rook/ceph:master"
 DEPLOYMENT_PLATFORM = 'AWS'
+TEST_NAMESPACE = 'ocs-qe'
```

Comment:
Does `TEST_NAMESPACE` really belong in `defaults.py`? This seems like it should just be a local variable in whatever script needs it.

Reply:
Nevermind this, reasoning was addressed in #147